### PR TITLE
semver version info on settings page and removal of next from package naming

### PR DESCRIPTION
**Pull request by mshariq-nerd**

### CODE REVIEW

File: src/app/components/layout/footer/footer.component.ts [hunk 1]
```diff
@@ -11,10 +9,19 @@ import * as moment from 'moment';
...
 
-  constructor(public router: Router) {}
+  constructor(public router: Router) { }
+
+  ngOnInit(): void {
+    if (this.git?.distance == 0) {
+      // released version
+      this.appVersion = `${this.git.version}-${this.git.hash}`
+    } else {
+      this.appVersion = this.git.raw.substr(0, this.git.raw.lastIndexOf('-'));
```
Comment (praveen-garg) [inline src/app/components/layout/footer/footer.component.ts:24]: can not we simply do,
 this.appVersion = this.git?.distance == 0 ? this.git.raw.replace( '-0-', '-') :  this.git.raw;

**Review by praveen-garg — COMMENTED** [review]
also handled the doc linking and remove next literal from package json, as we now have proper revision and git # mechanism.

**Review by praveen-garg — DISMISSED** [review]
lgtm!
@YashTatkondawar can we verify packaging with 1.9.3RC (dummy test) branch packaging and doc links considering tag etc.

**Review by YashTatkondawar — DISMISSED** [review]

**Review by praveen-garg — APPROVED** [review]
Thanks. Please get tested with a custom test tag package.

**Pull request merged**